templates/scl_util.epp

```diff
@@ -0,0 +1,5 @@
+<%- | String $util,
+      String $sclenvname
+| -%>
+#!/bin/bash
+scl enable <%= $sclenvname %> -- <%= $util %> $@
```

Comment:
In other scripts I've used `scl_source` and then `exec`. The benefit is that you have fewer processes. While most utils are short lived, it really makes you `ps auxf` easier to read. 
```suggestion
. scl_source enable <%= $sclenvname %>
exec <%= $util %> "$@"
```